File: Java/UberVan.java
package Java;

import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car {
    Map<String, ArrayList<String,Integer>> typeCarAccepted;
    ArrayList<String> seatMaterial;

    public UberVan(String license, Account driver,
    Map<String, ArrayList<String,Integer>> typeCarAccepted,
    ArrayList<String> seatMaterial) {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatMaterial = seatMaterial;
    }
}

File: Python/uberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted = []
    seatMaterail = []

    def __init__(self, license, driver,typeCarAccepted, seatMaterail):
        super(UberVan,self).__init__(license,driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatMaterail = seatMaterail
